tasks/release.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+/* eslint no-process-exit: 0, no-undef: 0, strict: 0 */
+'use strict';
+require( 'shelljs/global' );
+const colors = require( 'colors' );
```

Comment:
When I tried to run this for the first time I got the error `Error: Cannot find module 'colors'` and had to install `colors` using `npm` to be able to run it, so we should probably add it to `package.json` too.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1279/commits/e2657601e3bb1ce6786e5a1b93f60b7ce92a7162